components/automate-ui/src/app/pages/+compliance/shared/reporting/report-data.service.ts

```diff
@@ -39,14 +40,21 @@ export class ReportDataService {
   };
 
   constructor(
-    private statsService: StatsService
+    private statsService: StatsService,
+    private telemetryService: TelemetryService
   ) {}
 
   getReportingSummary(filters) {
     this.statsService.getSummary(filters)
       .subscribe(data => {
         this.reportingSummaryEmpty = this.isAllZeros(data.stats);
         this.reportingSummary = data;
+        // Data will give us nodes, platforms, profiles and environments
+        // We will also report filters so that we know when these counts are constrained.
+        if (data !== null) {
+          var summaryStats = data.stats
+          this.telemetryService.track("complianceCountsWithFilters", {summaryStats, filters})
```

Comment:
is the filter information useful? if not, should we send it when only start time and end time filters are applied?

Reply:
I wasn't sure, mostly because I am not super familiar with the compliance filters. Can start and end time be set to a super narrow window? That could skew things.

Reply:
start time is 10 days ago beg of day, end time is end of today utc. user can only modify end time, which is always end of today utc

Reply:
but the data that comes back for that specific endpoint is always "just today"

Reply:
(or, just whatever day is selected in the cal)